etc/ci/update_nightlies.sh

```diff
@@ -62,14 +62,15 @@ MASTER_COMMIT="$(git rev-parse HEAD)"
 
 git checkout -b gh-pages upstream/gh-pages || exit 1
 
-git rm -rf nightly || true
-rm -rf nightly || true
+OLD_NIGHTLIES="$(find nightly -type f -mtime +1)"
```

Comment:
You should probably also leave the `find` without the `exec` in there as well, so that travis will print out what files were found or not found, for debugging purposes.